frontend/src/scenes/teamLogic.tsx

```diff
@@ -34,17 +33,11 @@ export const teamLogic = kea<teamLogicType<TeamType>>({
             }
         },
         createTeamSuccess: () => {
-            let location = '/ingestion'
-            if (userLogic.values.user?.organization?.teams) {
-                for (const team of userLogic.values.user.organization.teams) {
-                    if (!team.is_demo && team.id !== values.currentTeam?.id) {
-                        /* If organization already has another non-demo project setup, take to settings, otherwise take to
-                        ingestion wizard */
-                        location = '/project/settings'
-                    }
-                }
+            if (values.currentTeam) {
```

Comment:
@macobo not sure if this makes sense anymore, a project will always have `completed_snippet_onboarding = False` after creation (per your latest change). Seems like the logic here could be as straightforward as redirecting to `/ingestion`?

Reply:
You're right

Reply:
Cool thanks will change myself, need to do one final tweak before merging